### PR TITLE
Database changes for Content Blocks

### PR DESCRIPTION
**Pull request by ebrett**
### Context
Standalone branch for release of database changes for content blocks

### Changes proposed in this pull request
I have split out the parts of main related to the database migrations and seeds. 

### Guidance to review
This is branch only for deployment to pre-prod and prod prior to the main 1.6 release (which also has these changes), so this PR will not be merged to main.  It is only for review purposes.

### CODE REVIEW

File: db/seeds/create_content_blocks.rb [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+* change colours, contrast levels and fonts
+* zoom in up to 300% without the text spilling off the screen
+* navigate the website using just a keyboard
+* navigate the website using speech recognition software
+* listen to the website using a screen reader (including the most recent versions of JAWS and NVDA)
+
+We’ve also made the website text as simple as possible to understand.
+
+# How accessible this website is
+
```
Comment (guy-roberts) [inline db/seeds/create_content_blocks.rb:38]: This should be a h2 

Darren noticed,  'How accessible this website is' within seed is down as h1 but should be h2.

**Review by guy-roberts — APPROVED** [review]

**Review by guy-roberts — APPROVED** [review]

File: db/seeds/create_content_blocks.rb (hunk 1 above)
Comment (ebrett) [reply db/seeds/create_content_blocks.rb:38]: ok, updated

**Review by dmallett1 — APPROVED** [review]
As prod

**Review by dmallett1 — APPROVED** [review]

**Review by guy-roberts — COMMENTED** [review]
Looks good to me

I agree that we should *never* run seed data on production.

What works for me is to load it onto another version and then create the Content Block on production by pasting in the markdown manually.

This only takes a few minutes

**Pull request closed without merging**